### ISSUE TITLE
Multiple ROCm packages [Out of Date]

### ISSUE THREAD

**Issue by itsagoodbrain** [arch4edu/arch4edu#138]
It seems the auto updates missed several rocm packages. Each should be updated to 4.3.1.

Sources on AUR all match this repo github.com/rocm-arch/rocm-arch

- rocm-libs
- rocm-utils
- hipfft
- rocffr

**Comment by hifi25nl** [comment]
Yes, I see that rocfft for example has not been updated to 4.3.1

**Comment by itsagoodbrain** [comment]
4.5.0 did just come out though. So it might make sense to wait and do that
update instead?

**Comment by itsagoodbrain** [comment]
I think for now, updating those packages would be good. I started another issue on rocm-arch to keep track of the version 4.5.0 release status.

**Comment by itsagoodbrain** [comment]
All the packages have been updated. Just waiting on the auto update to trigger...

**Comment by itsagoodbrain** [comment]
@petronny is there any chance the auto-updater isn't going to trigger? It's been a weed since the aur was updated.

**Comment by petronny** [comment]
Hi, most of the rocm packages should be updated now.
